ci-operator/templates/openshift/installer/cluster-launch-installer-upi-e2e.yaml

```
@@ -363,69 +385,248 @@ objects:
+        elif [[ "${CLUSTER_TYPE}" == "vsphere" ]]; then
+          mkdir -p /tmp/tf
 
-        # Copy sample UPI files
-        cp -r /var/lib/openshift-install/upi/${CLUSTER_TYPE}/* /tmp/tf
+          # Copy sample UPI files
+          cp -r /var/lib/openshift-install/upi/${CLUSTER_TYPE}/* /tmp/tf
 
-        # Create terraform.tfvars
-        export BOOTSTRAP_URL="http://${JOB_NAME_SAFE}-bootstrap-exporter-${NAMESPACE}.svc.ci.openshift.org/bootstrap.ign"
-        export MASTER_IGN=$(cat /tmp/artifacts/installer/master.ign)
-        export WORKER_IGN=$(cat /tmp/artifacts/installer/worker.ign)
+          # Create terraform.tfvars
+          export MASTER_IGN=$(cat /tmp/artifacts/installer/master.ign)
+          export WORKER_IGN=$(cat /tmp/artifacts/installer/worker.ign)
 
-        cat > /tmp/tf/terraform.tfvars <<-EOF
-          machine_cidr = "139.178.73.0/26"
+          cat > /tmp/tf/terraform.tfvars <<-EOF
+            machine_cidr = "139.178.73.0/26"
 
-          vm_template = "rhcos-latest"
+            vm_template = "rhcos-latest"
 
-          vsphere_cluster = "devel"
+            vsphere_cluster = "devel"
 
-          vsphere_datacenter = "dc1"
+            vsphere_datacenter = "dc1"
 
-          vsphere_datastore = "nvme-ds1"
+            vsphere_datastore = "nvme-ds1"
 
-          vsphere_server = "vcsa-ci.vmware.devcluster.openshift.com"
+            vsphere_server = "vcsa-ci.vmware.devcluster.openshift.com"
 
-          ipam = "139.178.89.254"
+            ipam = "139.178.89.254"
 
-          cluster_id = "${CLUSTER_NAME}"
+            cluster_id = "${CLUSTER_NAME}"
 
-          base_domain = "${BASE_DOMAIN}"
+            base_domain = "${BASE_DOMAIN}"
 
-          cluster_domain = "${CLUSTER_NAME}.${BASE_DOMAIN}"
+            cluster_domain = "${CLUSTER_NAME}.${BASE_DOMAIN}"
 
-          bootstrap_ignition_url = "${BOOTSTRAP_URL}"
+            bootstrap_ignition_url = "${BOOTSTRAP_URI}"
 
-          // Ignition config for the control plane machines. You should copy the contents of the master.ign generated by the installer.
-          control_plane_ignition = <<END_OF_MASTER_IGNITION
-          ${MASTER_IGN}
-          END_OF_MASTER_IGNITION
+            // Ignition config for the control plane machines. You should copy the contents of the master.ign generated by the installer.
+            control_plane_ignition = <<END_OF_MASTER_IGNITION
+            ${MASTER_IGN}
+            END_OF_MASTER_IGNITION
 
-          // Ignition config for the compute machines. You should copy the contents of the worker.ign generated by the installer.
-          compute_ignition = <<END_OF_WORKER_IGNITION
-          ${WORKER_IGN}
-          END_OF_WORKER_IGNITION
-        EOF
+            // Ignition config for the compute machines. You should copy the contents of the worker.ign generated by the installer.
+            compute_ignition = <<END_OF_WORKER_IGNITION
+            ${WORKER_IGN}
+            END_OF_WORKER_IGNITION
+          EOF
```

```suggestion
        EOF
```
This seems to break CI test

---

Fixed the here-doc indentation (I think ;) with 422d57100 -> ef62451ab.